onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -553,173 +552,134 @@ def sample_inputs_bernoulli_p_deterministic(op_info, device, dtype, requires_gra
         dtypes=common_dtype.floating_types_and(torch.bfloat16),
-        skips=(),
         sample_inputs_func=sample_inputs_max_pool2d_with_indices,
     ),
     opinfo_core.OpInfo(
-        "nn.functional.max_pool3d_with_indices",
+        "ops.aten.max_pool3d_with_indices",
         aten_name="max_pool3d_with_indices",
-        supports_forward_ad=True,
-        supports_fwgrad_bwgrad=True,
         dtypes=common_dtype.floating_types_and(torch.bfloat16),
-        skips=(),
         sample_inputs_func=sample_inputs_max_pool3d_with_indices,
     ),
     # NOTE: torch.STFT has pre-padding and it's not supported by aten::stft
     # This custom OpInfo uses aten::stft directly.
     opinfo_core.OpInfo(
-        "aten.stft",
+        "ops.aten.stft",
         aten_name="stft",
-        op=torch.ops.aten.stft,
         dtypes=common_dtype.floating_and_complex_types_and(torch.half, torch.bfloat16),
         sample_inputs_func=sample_inputs_stft,
-        # Runs very slowly on slow gradcheck - alternatively reduce input sizes
-        gradcheck_fast_mode=True,
-        supports_forward_ad=True,
-        supports_fwgrad_bwgrad=True,
-        check_batched_forward_grad=False,
-        check_batched_grad=False,
-        check_batched_gradgrad=False,
-        supports_out=False,
-        gradcheck_nondet_tol=common_utils.GRADCHECK_NONDET_TOL,
     ),
     opinfo_core.OpInfo(
-        "aten.tensor.bool",
+        "ops.aten.tensor.bool",
         aten_name="tensor.bool",
-        op=torch.ops.aten.tensor.bool,
         dtypes=common_dtype.all_types_and(torch.half, torch.bfloat16),
         sample_inputs_func=sample_inputs_tensor_bool,
     ),
     opinfo_core.OpInfo(
-        "aten.tensor.float",
+        "ops.aten.tensor.float",
         aten_name="tensor.float",
-        op=torch.ops.aten.tensor.float,
         dtypes=common_dtype.all_types_and(torch.half, torch.bfloat16),
         sample_inputs_func=sample_inputs_tensor_float,
     ),
     opinfo_core.OpInfo(
-        "aten.tensor.int",
+        "ops.aten.tensor.int",
         aten_name="tensor.int",
-        op=torch.ops.aten.tensor.int,
         dtypes=common_dtype.all_types_and(torch.half, torch.bfloat16),
         sample_inputs_func=sample_inputs_tensor_int,
     ),
     opinfo_core.OpInfo(
-        # Unique ID used to connect
-        #  TorchLibOpInfo("aten.bernoulli.p", ...)
-        # and
-        #  opinfo_core.OpInfo("aten.bernoulli.p", ...)
-        "aten.bernoulli.p",
+        "ops.aten.bernoulli.p",
         aten_name="bernoulli.p",
-        op=torch.ops.aten.bernoulli.p,
         # dtypes can be a tuple of (torch.float, torch.double).
         dtypes=common_dtype.all_types(),
         sample_inputs_func=sample_inputs_bernoulli_p,
     ),
     opinfo_core.OpInfo(
         # Deterministic bernoulli sampling where p is either 0 or 1
-        "aten.bernoulli.p_deterministic",
-        aten_name="bernoulli.p",
+        "ops.aten.bernoulli.p_deterministic",
         op=torch.ops.aten.bernoulli.p,
```

Comment:
rm?

Reply:
Name is `ops.aten.bernoulli.p_deterministic` so we need to specify

Reply:
Because I created two cases `ops.aten.bernoulli.p` and `ops.aten.bernoulli.p_deterministic` so we can at least test some numbers. `ops.aten.bernoulli.p` tests only dtype and shape

Reply:
`ops.aten.bernoulli.p` is the op being tested.

Reply:
I see. So if op is unspecified, by default OpInfo searches for function `torch.<op_info_name>`.

Feels like we can extend existing op info w/ extra sample data? Hmm I recall you have PR in torch so if that's merged we don't need the special case here.

Reply:
But if we just want to do extensions locally, that should be feasible as well right?

Reply:
> But if we just want to do extensions locally, that should be feasible as well right?

It should be. `ops.aten.bernoulli.p` was not in op_db so I created one. It contains more valid inputs than `p_deterministic`, (p = 0.5) which are not deterministic.

We could use the `variant` field to create two opinfos of the same name and this issue should go away, but right now in our TorchLibOpInfo there is no way to pick specific variants. We are only able to skip/xfail on variants. So this can be improved in another pass.